lib/everypolitician/popolo/membership.rb

```diff
@@ -4,6 +4,10 @@ class Memberships < Collection; end
 
     class Membership < Entity
       attr_accessor :person_id, :on_behalf_of_id, :organization_id, :area_id, :role, :start_date, :end_date
+
+      def person
+        popolo.persons.find_by(id: person_id)
```

Comment:
Things like this instinctively scare me, as they have a tendency to end up being called inside a loop, and are hugely sub-optimal. But that's something that should be fixed at a slightly different level (e.g. in `find_by` itself).